web/sites/default/config/lightning_core.versions.yml

```diff
@@ -248,6 +248,8 @@ select2: 1.13.0
 sendgrid_integration: 1.2.0
 serialization: 0.0.0
 shortcut: 0.0.0
+simple_block: 1.3.0
```

Comment:
@RichardDavies , this file needs to be reverted. (Though I'm not really sure when lighting_core enable modules actually fires.)

Reply:
Removed.